### PR TITLE
chore: pipelines create DataSource

### PR DESCRIPTION
**Pull request by ksimon1**
**What this PR does / why we need it**:
chore: pipelines create DataSource

Pipelines now creates DataSource object, which is used by UI and is pointing to result DV/PVC. The DataSource will not be deleted when the PipelineRun is finished.

**Release note**:
```
chore: pipelines create DataSource

```

### CODE REVIEW

File: release/pipelines/windows-efi-installer/README.md [hunk 1]
```diff
@@ -52,17 +52,18 @@ After the ISO is modified it creates a new VirtualMachine which boots from the m
...
-5. `create-vm` Task creates a VirtualMachine. A DataVolume with the Windows source ISO will be attached as CD-ROM and a second empty DataVolume will be used as installation destination. A third DataVolume with the virtio-win ISO will also be attached (Pipeline parameter `virtioContainerDiskName`). The VirtualMachine has to be created in the same namespace as the DataVolume with the ISO file. In case you would like to run the VirtualMachine in a different namespace, both Datavolumes have to be copied to the same namespace as the VirtualMachine.
-6. `wait-for-vmi-status` Task waits until the VirtualMachine shuts down.
-7. `cleanup-vm` deletes the installer VirtualMachine and all of its DataVolumes.
-8. The output artifact will be the `baseDvName`/`baseDvNamespace` DataVolume with the basic Windows installation. It will boot into the Windows OOBE and needs to be setup further before it can be used.
-9. `delete-imported-configmaps` deletes imported ConfigMaps.
+3. `create-datasource-root-disk` Task creates DataSource object which is used by UI and links PVC created in `create-vm-root-disk` step.
```
Comment (jcanocan) [inline release/pipelines/windows-efi-installer/README.md:59]: Same as above.

File: templates-pipelines/windows-customize/README.md [hunk 2]
```diff
@@ -21,16 +21,17 @@ The provided reference ConfigMap (`windows-sqlserver`) boots Windows 10, 11 or W
...
 2. `copy-vm-root-disk` Task copies PVC defined in `sourceDiskImageName` and `sourceDiskImageNamespace` parameters.
-3. `create-vm` Task creates a VirtualMachine called `windows-customize-*` from the base DataVolume and with the customize ConfigMap attached as a CD-ROM (Pipeline parameter `customizeConfigMapName`). The VirtualMachine has to be created in the same namespace as the source DataVolume.
-4. `wait-for-vmi-status` Task waits until the VirtualMachine shuts down.
-5. `cleanup-vm` deletes the installer VirtualMachine (also in case of failure of the previous Tasks).
-6. The output artifact will be the `win*-customized` DataVolume with the customized Windows installation. It will boot into the Windows OOBE and needs to be setup further before it can be used (depends on the applied customizations).
-7. The `windows11-unattend` ConfigMap can be used to boot the VirtualMachine into the Desktop (depends on the applied customizations).
-8. `delete-imported-configmaps` deletes imported ConfigMaps.
+3. `create-datasource-root-disk` Task creates DataSource object which is used by UI and links PVC created in `copy-vm-root-disk` step.
```
Comment (jcanocan) [inline templates-pipelines/windows-customize/README.md:28]: Same as above.

File: templates-pipelines/windows-efi-installer/README.md [hunk 3]
```diff
@@ -52,17 +52,18 @@ After the ISO is modified it creates a new VirtualMachine which boots from the m
...
-5. `create-vm` Task creates a VirtualMachine. A DataVolume with the Windows source ISO will be attached as CD-ROM and a second empty DataVolume will be used as installation destination. A third DataVolume with the virtio-win ISO will also be attached (Pipeline parameter `virtioContainerDiskName`). The VirtualMachine has to be created in the same namespace as the DataVolume with the ISO file. In case you would like to run the VirtualMachine in a different namespace, both Datavolumes have to be copied to the same namespace as the VirtualMachine.
-6. `wait-for-vmi-status` Task waits until the VirtualMachine shuts down.
-7. `cleanup-vm` deletes the installer VirtualMachine and all of its DataVolumes.
-8. The output artifact will be the `baseDvName`/`baseDvNamespace` DataVolume with the basic Windows installation. It will boot into the Windows OOBE and needs to be setup further before it can be used.
-9. `delete-imported-configmaps` deletes imported ConfigMaps.
+3. `create-datasource-root-disk` Task creates DataSource object which is used by UI and links PVC created in `create-vm-root-disk` step.
```
Comment (jcanocan) [inline templates-pipelines/windows-efi-installer/README.md:59]: Same as above.

**Review by jcanocan — COMMENTED** [review]
This is great! Thanks a lot @ksimon1!

**Review by 0xFelix — APPROVED** [review]
/lgtm

**Pull request merged**